Adding storage gloss terms (#335)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

*See also*: xref:bare-metal-n[bare metal (noun)]

[[bare-metal-service]]
==== image:images/yes.png[yes] Bare Metal service (noun)

> *Description*: The Bare Metal service is a Red Hat OpenStack Platform product that provisions bare metal machines.

Reviewer: ```suggestion
*Description*: The _Bare Metal service_ is a Red Hat OpenStack Platform product that provisions bare metal machines.
```
Per ISG: Words and phrases that are being defined or explained should use italic text: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting.

*See also*:

// RHEL: Added "In Red Hat Enterprise Linux,"

[[block-device]]
==== image:images/caution.png[with caution] block device (noun)

> *Description*: A physical storage device that supports reading and writing data in groups (blocks) of bytes, in any order. Do not confuse for "block storage" (cloud storage) or "block volume" (OpenShift Virtualization).

Reviewer: ```suggestion
*Description*: A _block device_ is a physical storage device that supports reading and writing data in groups (blocks) of bytes in any order. Do not confuse this term with _block storage_ (cloud storage) or _block volume_ (OpenShift Virtualization).
```
Per ISG: Words and phrases that are being defined or explained should use italic text: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting.

*Use it*: with caution

*Incorrect forms*: block storage, block volume

*See also*:

[[block-storage]]
==== image:images/caution.png[with caution] block storage (noun)

> *Description*: Block storage, as distinct from file storage and object storage, breaks data into chunks that are arbitrarily organized and distributed. These chunks can then be further partitioned and treated as individual storage devices. Do not confuse for "block device" (generic) or "block volume" (OpenShift Virtualization).

Reviewer: ```suggestion
*Description*: _Block storage_, as distinct from file storage and object storage, breaks data into chunks that are arbitrarily organized and distributed. These chunks can then be further partitioned and treated as individual storage devices. Do not confuse for _block device_ (generic) or _block volume_ (OpenShift Virtualization).
```

*Use it*: with caution

*Incorrect forms*: block device, block volume

*See also*:

[[block-volume]]
==== image:images/caution.png[with caution] block volume (noun)

> *Description*: A physical volume that supports `Block` as opposed to `Filesystem` data formats. Block data are raw chunks of data that are not compatible with any predefined file system mechanism such as NFS. Do not confuse for "block device" (generic) or "block storage" (cloud storage).

Reviewer: ```suggestion
*Description*: A _block volume_ is a physical volume that supports `Block` as opposed to `Filesystem` data formats. Block data are raw chunks of data that are not compatible with any predefined file system mechanism such as NFS. Do not confuse this term with _block device_ (generic) or _block storage_ (cloud storage).
```

*See also*:

[[boot-source]]
==== image:images/yes.png[yes] boot source (noun)

> *Description*: A system image containing a bootable operating system (OS) and all of the configuration settings for the OS, such as drivers. Boot sources can be used to create virtual machine templates with specific configurations. These templates can be used to create any number of available virtual machines.

Reviewer: ```suggestion
*Description*: A _boot source_ is a system image containing a bootable operating system (OS) and all of the configuration settings for the OS, such as drivers. Boot sources can be used to create virtual machine templates with specific configurations. These templates can be used to create any number of available virtual machines.
```

*See also*: xref:Bps[Bps]

[[brick]]
==== image:images/yes.png[yes] brick (noun)

> *Description*: Bricks are the basic unit of storage in Red Hat Gluster Storage. A brick is an exported directory on a server that is in a trusted storage pool.

Reviewer: ```suggestion
*Description*: A _brick_ is the basic unit of storage in Red Hat Gluster Storage. A _brick_ is an exported directory on a server that is in a trusted storage pool.
```
